src/Cello/problem_MethodOutput.cpp

```diff
@@ -396,7 +399,7 @@ void MethodOutput::compute_done (Block * block)
 void Block::r_method_output_done(CkReductionMsg *msg)
 {
   delete msg;
-  MethodOutput * method = static_cast<MethodOutput*> (this->method());
+  //MethodOutput * method = static_cast<MethodOutput*> (this->method());
```

Comment:
I commented this out because it was an unused variable. I'm not sure if there was any reason to keep it... If you think I should delete it, I can do that

Reply:
```suggestion
```